pkg/plugin/clientmgmt/logrus_adapter.go

```diff
@@ -30,7 +30,7 @@ const pluginNameField = "pluginName"
 // delegates all calls to a logrus logger.
 type logrusAdapter struct {
 	impl  logrus.FieldLogger
-	level logrus.Level
+	level hclog.Level
```

Comment:
Why replace logrus as part of this?

Reply:
On switching to using go modules, I could not get `hclog` library version w/o the `SetLevel` method  in the interface definition. Now, to use our logrus adapter as a hclog logger I had to implement 
```
func (l *logrusAdapter) SetLevel(level hclog.Level) {
	l.level = level
}
```
Now as I type this, I think I can I can do something like
```
hclog.LevelFromString(logLevel.String())
```

here to keep the changes minimal.

Reply:
I take that back. There is no way to convert a `hclog.Level` to `logrus.Level` even though they are are both `int32` and `uint32`.